.travis.yml

```diff
@@ -23,6 +23,13 @@ matrix:
       env: PANDAS_VERSION_STR="NONE"
     - python: 2.7
       env: PANDAS_VERSION_STR="NONE"
+    - python: 3.7
+      sudo: true
+      dist: xenial
+    - python: 3.7
+      sudo: true
+      dist: xenial
+      env: PANDAS_VERSION_STR="NONE"
```

Comment:
This is the simplest change that will work, but the overall situation is kind of silly. The only reason `sudo: true, dist: xenial` is needed is to get travis's version of 3.7, but we don't use travis's version of 3.7 for anything at all. We request it, force Travis to install it, and then we immediately throw it away and switch to conda. (And this is true for all the Python versions, just it becomes more obviously silly at the point where we start adding workarounds for limitations of Travis's Python...)

We'd be better off using `language: none`, and setting `PYTHON_VERSION` as an env-var... that's a larger change though.

Reply:
Agreed. I went ahead and pushed how I think you could do it without the hack. Honestly, its not much work and not that much of a larger change, so i'd suggest using it. You'll likely want to squash all my commits.